setup.py

```diff
@@ -95,7 +97,7 @@ def build_extensions(self):
                       ],
     setup_requires=['numpy', 'cython'],
```

Comment:
```suggestion
    setup_requires=['numpy', 'cython', 'setuptools_scm'],
```
I think you need to add this, if you want to use setuptools_scm. (but why are we using it?)

Reply:
I was using it just to get my installation working, but using setup from numpy fixed it, so there's no longer need for this.